DeveloperExcuses/View/OnelinerView.swift

```diff
@@ -68,8 +68,19 @@ open class OnelinerView: ScreenSaverView {
         preconditionFailure("`fetchOneline` must be overridden")
     }
     
+    open var onelineFontSize: Double {
+        let size = UserDefaults.standard.double(forKey: .onelineFontSize)
+        if size == 0 {
+            let defaultSize = 24.0
+            UserDefaults.standard.set(defaultSize, forKey: .onelineFontSize)
```

Comment:
I'd sync the `UserDefaults` here.

```suggestion
            UserDefaults.standard.set(defaultSize, forKey: .onelineFontSize)
            UserDefaults.standard.synchronize()
```